### DISCUSSION TITLE
bias_steps.take_bgmap(S,cfg) leaves a 0.29V bias on all bias lines

### DISCUSSION THREAD

**Discussion by samdayweiss** [simonsobs/sodetlib#399] [General]
Hi All, we've noticed on TSAT that `bias_steps.take_bgmap()` sometimes (meaning, not consistently, but it has happened 4-5 times to us now on different slots) leaves a 0.3V bias on the all bias lines after running. Is there no routine to check that all biases are set to 0?

Thanks!

**Comment by msilvafe** [comment]
It looks like this is the way the function is written this way, with a `dc_voltage` parameter that is default to 0.3V: https://github.com/simonsobs/sodetlib/blob/56d04ce97fc90abcadcba07b1bbd8841cacf74be/sodetlib/operations/bias_steps.py#L1060 The doc string does not actually described why that dc_voltage is needed but you could just try running that function but passing dc_voltage = 0 instead. We may consider modifying the function to set the dc_voltage to 0 at the end of the function anyway but maybe @jlashner can recall why we added that argument in the first place?

**Reply by jlashner** [reply]
Ah yes I forgot that I did that. I think the non-zero DC voltage makes the math of computing DC params easier, so you don't get infinities places like here: https://github.com/simonsobs/sodetlib/blob/56d04ce97fc90abcadcba07b1bbd8841cacf74be/sodetlib/operations/bias_steps.py#L657-L658

**Reply by msilvafe** [reply]
In that case should we pr a change that zeros it out at the end?

**Reply by jlashner** [reply]
ya i think that's valid

**Comment by msilvafe** [comment]
Ok, addresses in PR #400.
